src/ResourceManager/Relay/Commands.Relay/Cmdlets/Namespace/NewAzureRelayNamespaceAuthorizationRules.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Relay.Models;
+using Microsoft.Azure.Management.Relay.Models;
+using System.Management.Automation;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Relay.Commands.Namespace
+{
+    /// <summary>
+    /// 'New-AzureRmRelayNamespaceAuthorizationRule' Cmdlet creates a new Relay Namespace AuthorizationRule
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, RelayNamespaceAuthorizationRuleVerb, SupportsShouldProcess = true), OutputType(typeof(AuthorizationRuleAttributes))]
```

Comment:
Same comment - AuthorizationRule cmdlets should be centralized, not individual for each object